src/Network/NetRc.hs

```diff
@@ -238,10 +254,14 @@ hostEnt = do
     -- pval := ((account|username|password) WS+ <value>)
     pval = hlp "login"    PValLogin   <|>
            hlp "account"  PValAccount <|>
-           hlp "password" PValPassword
+           hlpPassword
       where
         hlp tnam cons = P.try (P.string tnam) *> wsChars1 *>
                         (cons <$> tok P.<?> (tnam ++ "-value"))
+        hlpPassword   = P.try (P.string "password") *> wsChars1 *>
+                        (PValPassword <$> password P.<?> "password-value")
```

Comment:
Would it be possible to parametrize `hlp` by the parser (`tok` or `password`) rather than duplicating it into `hlpPassword`?